microk8s-resources/wrappers/apiservice-kicker

```diff
@@ -45,4 +46,17 @@ do
             restart_attempt=0
         fi
     fi
+
+    # If no local-registry-hosting documentation has been installed,
+    # install a help guide that points to the microk8s registry docs.
+    if [ $installed_registry_help -eq 0 ] &&
```

Comment:
I put this code in the apiservice-kicker, because it needs to happen after the apiservice comes up. 

This seemed like the best fit for that kind of a life-cycle hook in the current code (even if the fit isn't that great)

But I'm completely open to suggestions on better places to put it!

Reply:
Indeed we do not have a place to put any maifests we want to apply at boot time.

As you said, it is good enough for now. We will explore alternatives if this causes problems.